docs/infrastructure/microk8s-quickstart.md

```diff
@@ -45,6 +45,12 @@ Install MicroK8s on the machine:
     microk8s.status --wait-ready
     ```
 
+4.  Disable the high-availability option:
+
+    ```bash
+    microk8s.disable ha-cluster
+    ```
```

Comment:
Here it would be interesting to provide a little bit more context (a recommendation or a justification) for disabling the high availability clustering.

For example, something like:

> If you're running microk8s using a single node, disable the high availability clustering to <...>:

Also, is high availability clustering enabled by default when customers install Codacy? By [looking at the docs](https://microk8s.io/docs/high-availability) it seems that it's only enabled in clusters with three or more nodes. Do customers installing Codacy really need to perform this step? :thinking: @Ngwerume

Reply:
I suggest:

* mentioning the condition under which the customer should disable clustering
* keeping the reason generic, such as "improved performance"

Let me know if this is OK.

```suggestion
4.  If you're running MicroK8s using a single node, disable high-availability clustering for improved performance:

    ```bash
    microk8s.disable ha-cluster
    ```
```

Reply:
Thanks for clarifying @Ngwerume! :+1: I think that @nicklem's new proposal is spot on. :raised_hands: